scripts/linters/custom_eslint_checks/rules/priv.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Add lint check to ensure that all private functions
+ *      /variables names
+ *      start with _,and that all functions/variables that
+ *      start with _ are tagged as private.
+ */
+
+'use strict';
+
+const console = require("node:console");
+const { isVariableDeclaration } = require("typescript");
+const { global } = require("yargs");
+
+// TODO(#10479): Implement this rule using the nodes instead of tokens.
+
+module.exports = {
+    meta: {
+        type: "suggestion",
+        docs: {
+          description: (
+            'Add lint check to ensure that all private functions/' +
+            'variables names start with _, and that all functions/variables' +
+            'that start with _ are tagged as private'),
+            category: "Possible Errors",
+            recommended: true,
+            url: "https://eslint.org/docs/rules/no-extra-semi"
+        },
+        fixable: "code",
+        schema: [], 
+        messages: {
+          expectedPriv: 'Functions/Variables starting with _ should be private',
+          expected_ : 'Private functions/variables should start with _'
+        }
+    },
+    create: function(context) {
+      var checkPrivWith_ = function(testVarNode, testVar) {
+        if (!testVar.startsWith('_') && testVar.getScope() != 'global') {
```

Comment:
@shubhamcodez Can you please use `ClassProperty[accessibility="private'"]` selector? I'm expecting this selector to work, let me know if you ned any help! :)

Reply:
> @shubhamcodez Can you please use `ClassProperty[accessibility="private'"]` selector? I'm expecting this selector to work, let me know if you ned any help! :)

Okay @DubeySandeep. I am unfamiliar with Classproperty selector, I didn't find anything on it, can you help me out there.
 Also, How do I run tests on the implementation.

Reply:
You can learn how to write rules in eslint [here](https://eslint.org/docs/developer-guide/working-with-rules), and you can run tests using scripts.run_custom_eslint_tests.

I would also suggest you start with other starter issues and work on the linter issue after completing a few starter issues. (As we have removed the "good first issue" label from #8423)